### PR TITLE
Unified Syscalls

### PR DESCRIPTION
**Pull request by rennergade**
This is the ultimate merge of the past several months work of getting rid of the alternate "lind_syscalls" pathways in libc and nacl.

Let's all give it a final review before we merge it in.

### CODE REVIEW

**Review by jesings — APPROVED** [review]
This is mostly boiler-plate aside from the 2 lines that should be fixed, looks merge-ready

File: elf/Versions
Comment (kapkic) [reply elf/Versions:174]: I was thinking the same, if they are not going to be implemented let's remove it.

File: sysdeps/nacl/nacl_syscalls.h
Comment (kapkic) [inline sysdeps/nacl/nacl_syscalls.h:138]: We can remove the empty lines between consecutive call numbers.

**Review by youngfalcon — APPROVED** [review]

**Comment by rennergade** [comment]
Ok this seems good to go once nacl is merged.

**Pull request merged**
